config/application.rb

```diff
@@ -98,6 +98,8 @@ class Application < Rails::Application
     ActiveSupport.to_time_preserves_timezone = false
 
     ### End Rails 5.0 config flags
+
+    config.youtube = YAML.load_file('config/youtube.yml')
```

Comment:
I went down that path initially, but switched to simply loading the YAML directly because this doesn't seem like a setting that needs to be different in development/test/production, and `config_for` requires that we supply settings per-environment.

Reply:
My thinking for using a YAML file was that we might use a different setting in test/dev environments, and also that it would lay the groundwork for eventually moving the rest of the Youtube settings out of `config/secrets.yml` and into `config/youtube.yml`. However, we haven't identified a need to do either of those things at this point, so I think we should keep it simple and just use a constant and no config file at all.

Reply:
OK.